staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go

```diff
@@ -852,6 +854,10 @@ func (s *GenericAPIServer) InstallAPIGroups(apiGroupInfos ...*APIGroupInfo) erro
 	}
 
 	for _, apiGroupInfo := range apiGroupInfos {
+		if len(apiGroupInfo.PrioritizedVersions) == 0 {
+			klog.Warningf("Skipping API %v because it has no PrioritizedVersions.", *apiGroupInfo)
+			continue
+		}
```

Comment:
drop this change, this is a programming error that should be fixed, not warned about and ignored